uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Listener/Form/Admin/Extensions.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CRM_HRCore_Listener_Form_Admin_Extensions extends CRM_HRCore_Listener_AbstractListener {
```

Comment:
so here too, instead of CRM_HRCore_Listener_Form_Admin_Extensions use CRM_HRCore_HookListener_Form_Admin_Extensions